osdep/subprocess-win.c

```diff
@@ -402,6 +403,7 @@ void mp_subprocess2(struct mp_subprocess_opts *opts,
     // Specify which handles are inherited by the subprocess. If this isn't
     // specified, the subprocess inherits all inheritable handles, which could
     // include handles created by other threads. See:
+    // https://web.archive.org/web/20121127222200/
     // http://blogs.msdn.com/b/oldnewthing/archive/2011/12/16/10248328.aspx
```

Comment:
Could link to the new place where this post reside https://devblogs.microsoft.com/oldnewthing/20111216-00/?p=8873

Reply:
Agreed.